src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3740,7 +3752,42 @@ class CommonVisitor : public AST::BaseVisitor<Struct> {
             }
             return final_result;
         } else {
+            ASR::expr_t *index = nullptr;
             this->visit_expr(*m_slice);
+            if (ASR::is_a<ASR::Const_t>(*type))
+            {
```

Comment:
Minor: Formatting suggestion
```suggestion
            if (ASR::is_a<ASR::Const_t>(*type)) {
```